docs/guide_rate.md

```diff
@@ -14,6 +18,8 @@ The module can then be imported into a python session/notebook. The main class u
 from statstruk import ratemodel
 ```
 
+
```

Comment:
Can you remove the blank lines here? I think it is causing one of the checks to fail.